pldoc/_design_elements/icons.md

```diff
@@ -37,148 +37,113 @@ info:               Icons are used to clearly and visually represent some inform
 
     <div class="example example-icon">
         <div class="icon-display">
-            <span class="icon-fallback icon-fallback-text">
-                <span class="icon icon-error" aria-hidden="true"></span>
-                <span class="text">Error</span>
-            </span>
+            <span class="icon fa fa-exclamation-circle" aria-hidden="true"></span>
+            <span class="sr-only">Error</span>
         </div>
         <div class="use-description">
             <p>Icon with text fallback</p>
         </div>
     </div>
 
-    <div class="example example-icon">
-        <div class="icon-display">
-            <span class="icon-fallback icon-fallback-img">
-                <span class="icon icon-warning" aria-hidden="true"></span>
-                <span class="text">Warning</span>
-            </span>
-        </div>
-        <div class="use-description">
-            <p>Icon with image fallback</p>
-        </div>
-    </div>
 </div>
 
 <h3 class="hd-6 example-set-hd">Media</h3>
```

Comment:
I would also add a paragraph to the opening description saying that we use Font Awesome, and provide a link to its icon set. It would probably also be worth saying something about accessibility and maybe linking to http://fontawesome.io/accessibility/